diff --git a/openwrt/build.sh b/openwrt/build.sh
index 8c2d1e7..5e0dd67 100644
--- a/openwrt/build.sh
+++ b/openwrt/build.sh
@@ -256,6 +256,8 @@ curl -sO https://$mirror/openwrt/scripts/02-prepare_package.sh
 curl -sO https://$mirror/openwrt/scripts/03-convert_translation.sh
 curl -sO https://$mirror/openwrt/scripts/04-fix_kmod.sh
 curl -sO https://$mirror/openwrt/scripts/05-fix-source.sh
+curl -sO https://$mirror/openwrt/scripts/10-customize-config.sh
+curl -sO https://$mirror/openwrt/scripts/11-fix-vendor.sh
 curl -sO https://$mirror/openwrt/scripts/99_clean_build_cache.sh
 chmod 0755 *sh
 [ "$(whoami)" = "runner" ] && group "patching openwrt"
@@ -263,6 +265,10 @@ bash 00-prepare_base.sh
 bash 02-prepare_package.sh
 bash 03-convert_translation.sh
 bash 05-fix-source.sh
+bash 10-customize-config.sh
+if [ "$platform" = "x86_64" ] || [ "$platform" = "armv8" ]; then
+    bash 11-fix-vendor.sh
+fi
 if [ "$platform" = "rk3568" ] || [ "$platform" = "rk3399" ] || [ "$platform" = "x86_64" ] || [ "$platform" = "bcm53xx" ] || [ "$platform" = "armv8" ]; then
     bash 01-prepare_base-mainline.sh
     bash 04-fix_kmod.sh
